docs/integrations/sources/gitlab-migrations.md

```diff
@@ -1,5 +1,64 @@
 # Gitlab Migration Guide
 
+
+## Upgrading to 4.0.0
+
+In this release, a format of stream state has been changed due to migration to low-code.
```

Comment:
We're continuously striving to enhance the quality and reliability of our connectors at Airbyte.  As part of our commitment to delivering exceptional service, we are transitioning Source Gitlab from the Python Connector Development Kit (CDK) to our a new low-code framework improving maintainability and reliability of the connector. However, due to differences between the Python and low-code CDKs, this migration constitutes a breaking change.

Reply:
There was a typo:

We're continuously striving to enhance the quality and reliability of our connectors at Airbyte. As part of our commitment to delivering exceptional service, we are transitioning Source Gitlab from the Python Connector Development Kit (CDK) to our new low-code framework improving maintainability and reliability of the connector. However, due to differences between the Python and low-code CDKs, this migration constitutes a breaking change.

Reply:
Updated